qwerky_vllm_models/modeling/layers.py

```
@@ -17,137 +17,149 @@
-# =============================================================================
 
 class MLP(nn.Module):
     """MLP layer with fused gate+up projection and SiluAndMul activation."""
 
-    def __init__(self, d_model: int, intermediate_size: int, hidden_act: str = "silu",
-                 prefix: str = "", quant_config=None):
+    def __init__(
+        self,
+        d_model: int,
+        intermediate_size: int,
+        hidden_act: str = "silu",
```

Ruff (0.15.4)

[warning] Unused method argument: `hidden_act`

(ARG002)